### ISSUE TITLE
Widget Feed Support

### ISSUE THREAD

**Issue by pr0927** [Ashinch/ReadYou#89]
Wanted to request addition of a widget feature for a newsfeed of articles. Something like the Nextcloud News app (but one which actually opens to the correct article upon tapping...).

I think this is the single aspect I struggle to find for RSS news apps, and it's what generally holds me back from adopting. I have no idea how popular this request would be, but I'm sure I wouldn't be the only one who'd love to see it. Love what I see in this app otherwise!

**Comment by surfaceflinger** [comment]
Widget with a nice material you design could easily replace google feed for users with no gapps

**Comment by pr0927** [comment]
Yeah this would totally automatically make this my RSS reader of choice - the singular bane of my Android experience is not having a good FOSS RSS widget. Using Neo Launcher, so I could put it on the "left" side and not bother with any Google Now page nonsense that way too.

**Comment by lxwulf** [comment]
Also I fully agree with you guys! Would really appreciate this feature!

**Comment by Ansem93** [comment]
No widget is basically the reason why it's not my main app.
Widget for rss feed should be a must!

**Comment by aksh1618** [comment]
Would love this! I'm using Inoreader right now but want to switch away to OSS-only tools, and the main thing I miss is the widget that comes with Inoreader. I might take a shot at implementing it myself at some point if I get time, but in the meanwhile if someone else wants to have a go, they can use Inoreader's implementation as a reference. My main requirement would be to at least support selecting specific groups as a list widget.

**Comment by shuvashish76** [comment]
> use Inoreader's implementation as a reference

FOSS project [NewsBlur](https://github.com/samuelclay/NewsBlur) has nice widget customization options e.g. exactly which sites to show on widget, sort/order, transparency effects etc. Take a look at their implementation.

<details><summary>screenshot</summary>
<p>

![Screenshot](https://github.com/Ashinch/ReadYou/assets/48226312/a01ca110-2abd-4179-8d7d-204ea18041df)

</p>
</details>

**Comment by dubsflyt** [comment]
The card widget is 2x2 only, can a 4x2 version be added or more ideally it be resizable?
